0-bootstrap/README-Jenkins.md

```diff
@@ -281,21 +273,30 @@ You arrived to these instructions because you are using the `jenkins_bootstrap`
+    1. Test that your Jenkins Master can deploy a [pipeline](https://www.jenkins.io/doc/book/pipeline/getting-started/) to the Jenkins Agent located in the `prj-cicd` project (you can test this by running with a simple `echo "Hello World"` pipeline build).
 
-### IV. Configure the Git repositories and Multibranch Pipeline in your Jenkins Master
+### IV. Configure the Git repositories and Multibranch Pipelines in your Jenkins Master
 
-- **Note:** this section is considered out of the scope of this document, since there are multiple options on how to configure the Git repositories and **Multibranch Pipeline** in your Jenkins Master. Here we provide some guidance that you should keep in mind while completing this step. Visit the [Jenkins website](http://jenkins.io) for more information, there are plenty of Jenkins Plugins that could help with the task.
+- **Note:** this section is considered out of the scope of this document. Since there are multiple options on how to configure the Git repositories and **Multibranch Pipeline** in your Jenkins Master, here we can only provide some guidance that you should keep in mind while completing this step. Visit the [Jenkins website](http://jenkins.io) for more information, there are plenty of Jenkins Plugins that could help with the task.
+    - You need to configure a **"Multibranch Pipeline"**. Note that the `Jenkinsfile` and `tf-wrapper.sh` files use the `$BRANCH_NAME` environment variable. **the `$BRANCH_NAME` variable is only available in Jenkins' Multibranch Pipelines**.
 
-1. Create Multibranch pipelines for your new repos (`YOUR_NEW_REPO-1-org, YOUR_NEW_REPO-2-environments, YOUR_NEW_REPO-3-networks, YOUR_NEW_REPO-4-projects`).
-    - **DO NOT configure an automatic pipeline for your `YOUR_NEW_REPO-0-bootstrap` repository**
+1. In your Jenkins Master Web UI, **create Multibranch Pipelines only for the following repositories:**
+    ```
+    YOUR_NEW_REPO-1-org
+    YOUR_NEW_REPO-2-environments
+    YOUR_NEW_REPO-3-networks
+    YOUR_NEW_REPO-4-projects
+    ```
+    - **Reminder: DO NOT configure an automatic pipeline for `YOUR_NEW_REPO-0-bootstrap`**
 
 1. Assuming your new Git repositories are private, you may need to configure new credentials In your Jenkins Master web UI, so it can connect to the repositories.
-1. You will need to configure a "**Multibranch Pipeline**" for each one of the repositories. Note that the `Jenkinsfile` and `tf-wrapper.sh` files use **the `$BRANCH_NAME` environment variable, which is only available in Multibranch Pipelines** in Jenkins.
+
 1. You will also want to configure automatic triggers in each one of the Jenkins Multibranch Pipelines, unless you want to run the pipelines manually from the Jenkins Web UI after each commit to your repositories.
 
+1. You can now move to the instructions in the step [1-org](../1-org/envs/shared/README.md).
```

Comment:
The correct link is `../1-org/README.md`

Reply:
Done.